hs_core/views/resource_api.py

```diff
@@ -624,7 +625,7 @@ def delete(self, _, pk):
         raise NotImplemented()
 
     def publish_resource(self, pk):
-        authorize(self.request, pk, full=True)
+        authorize(self.request, pk, needed_permission=Action_To_Authorize.SET_RESOURCE_FLAG)
```

Comment:
@pkdash Again, SET_RESOURCE_FLAG may be sufficient as the action permission for publishing a resource, but want to raise this question for you to check just to make sure.